packages/framework-info/src/frameworks/angular.json

```diff
@@ -22,5 +22,10 @@
     "dark": "/logos/angular/default.svg"
   },
   "env": {},
-  "plugins": []
+  "plugins": [
+    {
+      "packageName": "@netlify/angular-runtime",
+      "condition": { "minNodeVersion": "17.0.0" }
```

Comment:
Is this what you meant to do?

Reply:
nope, it isn't! I meant to do "from angular v17 on, use this". gonna look at how to do that. good catch!´

Reply:
It's changed since I last worked on it, so I'm afraid I can't help there!

Reply:
implemented `minFrameworkVersion` in e6298e96cd0bda69e0cb789cefeebef1c2508ec3